compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -1963,8 +1963,29 @@ static LogicalResult setConvRootConfig(mlir::FunctionOpInterface entryPointFn,
   int64_t numTilingDims = parallelTileSizes.size();
   tileSizes.emplace_back(numTilingDims, 0);
 
+  // Set "scalable" flags
+  ScalableTileFlagsListType scalableTileFlags;
+  auto targetAttr = IREE::HAL::ExecutableTargetAttr::lookup(entryPointFn);
+  if (isAArch64(targetAttr) && hasAnySVEFeature(targetAttr) &&
+      clEnableScalableVectorization &&
+      isa<linalg::DepthwiseConv2DNhwcHwcOp>(convOp)) {
+
+    auto dims = linalg::inferConvolutionDims(convOp);
```

Comment:
Then this is unused? Do you plan to wait for the upstream change? It is okay to do it as a follow-up to me. If we are going to do it in a follow-up, please remove `dims`. Because it is unused.

Reply:
See the latest update :) After a discussion with @qedawkins (see https://github.com/llvm/llvm-project/pull/90057), I've realised that I misunderstood `inferConvolutionDims`. Yes, we can use it here :)

Reply:
oh thanks, sorry that I miss that!